src/scripts/readmeScript.ts

```diff
@@ -0,0 +1,35 @@
+import { COMMANDS } from '../commands/commands';
+import { readFile, writeFile } from 'fs';
+import { stripIndents } from 'common-tags';
+
+const flags = process.argv.slice(2);
+
+const commandTable = stripIndents`
+  ## Available Commands
+  | Prefix | Description |
+  | :-: | :-: |
+  ${COMMANDS.map(
+    (command) => '| ' + command.prefix + ' | ' + command.description + ' |\n'
+  ).join('')}
+`;
+
+// if there is an argument --autoUpdateReadme while running the command,
+// it will run this
+if (flags.includes('--autoUpdateReadme')) {
+  readFile('README.md', 'utf-8', (err, data) => {
+    if (err) throw err;
+
+    // Change the regex in this line to determine where you're going to put it at...
+    const results = data.replace(
+      /## Available Commands.*## Additional Information/gs,
+      commandTable
+    );
```

Comment:
```suggestion
const commandTable = stripIndents`
  ## Available Commands
  | Prefix | Description |
  | :-: | :-: |
  ${COMMANDS.map(
    (command) => '| ' + command.prefix + ' | ' + command.description + ' |\n'
  ).join('')}
  ##
`;

// if there is an argument --autoUpdateReadme while running the command,
// it will run this
if (flags.includes('--autoUpdateReadme')) {
  readFile('README.md', 'utf-8', (err, data) => {
    if (err) throw err;

    // Change the regex in this line to determine where you're going to put it at...
    const results = data.replace(
      /## Available Commands[^#]*##/gs,
      commandTable
    );
```

The current default RegEx will remove the `Additional Information` header. Changing the code to this will have the script find the Available Commands header, and replace all content until it finds the next `##`.